libkbfs/folder_branch_ops.go

```diff
@@ -2487,14 +2524,26 @@ func (fbo *folderBranchOps) finalizeGCOp(ctx context.Context, gco *GCOp) (
 	return fbo.notifyBatchLocked(ctx, lState, irmd)
 }
 
-func checkDisallowedPrefixes(name string, mode InitMode) error {
-	if mode == InitSingleOp {
-		// Allow specialized, single-op KBFS programs (like the kbgit
-		// remote helper) to bypass the disallowed prefix check.
-		return nil
-	}
+// CtxAllowNameKeyType is the type for a context allowable name override key.
+type CtxAllowNameKeyType int
+
+const (
+	// CtxAllowNameKey can be used to set a value in a context, and
+	// that value will be treated as an allowable directory entry
+	// name, even if it also matches a disallowed prefix.
+	CtxAllowNameKey CtxAllowNameKeyType = iota
```

Comment:
Can we mention in comment that the value to this key has to be `string` typed? Otherwise it may panic.